samples/msal-browser-samples/vue3-sample-app/README.md

```diff
@@ -0,0 +1,207 @@
+# MSAL.js 2.x + Vue 3 + TypeScript Sample
+
+## About this sample
+
+This sample demonstrates one way you can integrate the `@azure/msal-browser` package into your Vue 3 application using the [composition API](https://v3.vuejs.org/api/composition-api.html). It is not exhaustive and there may simpler or more complex solutions depending on your specific use case.
```

Comment:
Do we want to add a note that this will help but we're not currently supporting it at this time?  Otherwise, you'll likely have to take the bulk of the incoming questions on Vue.

Reply:
Thanks for the callout! Added.